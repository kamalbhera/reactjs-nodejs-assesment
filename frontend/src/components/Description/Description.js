import React from 'react';
import PropTypes from 'prop-types'
import './Description.scss';

const Description = ({ description}) => {
  return (
    <p className="description">{ description }</p>
  );
};

Description.propTypes = {
	description: PropTypes.string
};

export default Description;
