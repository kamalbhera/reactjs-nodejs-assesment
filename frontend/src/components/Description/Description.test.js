import React from 'react';
import { shallow } from 'enzyme';
import Description from './Description';

it('renders Description component', () => {
	const props = {
		description: "This is generic description",
	};
	const result = shallow(<Description { ...props }/>);
	const html = <p className="description">This is generic description</p>;
	expect(result).toMatchElement(html);
	expect(result).toMatchSnapshot();
});
