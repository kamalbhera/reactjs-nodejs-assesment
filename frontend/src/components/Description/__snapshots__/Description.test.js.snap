// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Description component 1`] = `
<p
  className="description"
>
  This is generic description
</p>
`;
